tasks/test.js

```diff
@@ -0,0 +1,45 @@
+module.exports = function(grunt) {
+
+  var Drupal = require('../lib/drupal')(grunt);
+  var Help = require('../lib/help')(grunt);
+
+  // Default to Drupal 8 unless Drupal 7 is explicitly defined.
+  var script = Drupal.majorVersion() == 7 ? './scripts/run-tests.sh' : './core/scripts/run-tests.sh';
+  var testLocation = Drupal.majorVersion() == 7 ? './sites/all/modules/custom' : './modules/custom';
+  var domain = process.env.GDT_DOMAIN || grunt.config('config.domain') || require('os').hostname();
+
+  grunt.config('test', {
+    behat: true,
+    drupal: {
+      command: []
+    }
+  });
+
+  grunt.config(['shell', 'testdrupal'], {
+    command: grunt.config.get('config.testing.drupal.command') || 'php ' + script + ' --verbose --color --concurrency 4 --directory ' + testLocation + ' --url ' + domain,
```

Comment:
There may be other reasons to customize the command (eg, arguments and options). Unless I'm missing something, the `grunt.shell` task this is extending doesn't appear to take options/args as separate config.